dev/io.openliberty.persistence.3.2.thirdparty/resources/org/eclipse/persistence/internal/localization/i18n/ExceptionLocalizationResource.nlsprops

```diff
@@ -99,10 +101,20 @@ cant_persist_detatched_object=Cannot PERSIST detached object, possible duplicate
 
 cant_refresh_not_managed_object=Cannot refresh unmanaged object: {0}.
 
+configured_pu_name_conflict=Cannot create configured persistence unit with name {0}. This name was found in custom persistence units.
+
 create_insertion_failed=Create insertion failed.
 
 criteria_no_constructor_found=An exception occurred while inspecting class {0} for a constructor using selection criteria types as arguments.  If this CriteriaQuery was not intended to be a constructor query please verify that the selection matches the return type.
 
+custom_pu_create_error=Cannot create custom persistence unit with name {0}.
```

Comment:
Global change to:

Cannot create a configured persistence unit with the name {0}.